extensions/notebook/src/test/book/book.test.ts

```diff
@@ -114,6 +118,15 @@ describe('BookTreeViewProviderTests', function () {
 			await fs.writeFile(markdownFile, '');
 		});
 
+		it('bookProviders should be initialized on extension activate', async () => {
+			let notebookExtension: vscode.Extension<any> = vscode.extensions.getExtension('Microsoft.notebook');
+			await notebookExtension.activate();
+			appContext = notebookExtension.exports.getAppContext();
```

Comment:
.activate() returns the API directly, you don't need to go through exports.

`const appContext = (await vscode.extensions.getExtension('Microsoft.notebook').activate()).getAppContext()`